models/app.models.js

```diff
@@ -1,10 +1,15 @@
-const db = require("../db/connection")
+const fs = require("fs/promises")
 
-function selectAllTopics () {
-    return db.query("SELECT * FROM topics")
-    .then((result) => {
-        return result.rows;
-    })
-}
 
-module.exports = selectAllTopics;
\ No newline at end of file
+// function selectAllApis () {
+//     return fs.readFile('./endpoints.json')
+//     .then((fileContents) => {
+//         const parsedFileContents = JSON.parse(fileContents)
+//         return parsedFileContents;
+//     })
+// }
+
```

Comment:
This file is doing nothing and shouldn't really be here.

Reply:
Was separating concerns and had to move things into their independent files. will remove it.